transformations/diverse_paraphrase/submod/submodopt_intent.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import scipy.linalg as la
+import pdb
+from collections import Counter
+import string
+
+from transformations.diverse_paraphrase.submod.submodular_funcs import distinct_ngrams
+
+
+class SubmodularOpt:
```

Comment:
This file seems to be not used. Correct me if I'm wrong

Reply:
You are correct, this was being used for some other results in the original paper. 
I'll remove this file.